libs/js-sdk/src/update-strategies/createManualUpdateStrategy.ts

```diff
@@ -44,13 +44,19 @@ export function createManualUpdateStrategy(
         onRequest() {
             return undefined
         },
-        updateAudiences(state, updatedAudiences) {
+        async updateAudiences(state, updatedAudiences) {
+            if (updatedAudiences.sort() === currentAudiences.sort()) {
+                // No need to update audiences
+                return Promise.resolve()
+            }
             currentAudiences = updatedAudiences
```

Comment:
I wonder if we could get rid of `currentAudiences` and just use `state.audiences` so there isn't the duplication? (am unsure)

Reply:
I think that is a good idea... Don't see a reason why it wouldn't be ok!